### ISSUE TITLE
Move UI settings

### ISSUE THREAD

**Issue by arthuro555** [4ian/GDevelop#1983]
## Description
UI settings are stored currently with the rest of the important. This is bad for 2 reasons:
1. It gets exported and adds unecessary size to the game
2. It generates many conflicts when using git for collaborations.

## Solution suggested
The user specific data (like UI settungs) should be stored in a special store that can be saved in a separate file (so that we can gitignore it) and won't get exported.

**Comment by 4ian** [comment]
@Bouh Could you create a card for it in the roadmap (in the IDE column)? Thanks! :)

**Comment by Bouh** [comment]
Yes it's done, even if i have loose the track of the roadmap, since some features are directly on github without be present on the roadmap^^

**Comment by arthuro555** [comment]
I just posted a 20$ bounty for this.

**Comment by nilaygit-10721** [comment]
please assign me this issue?

**Comment by prafulaa** [comment]
Taking this on — I've been looking into how `uiSettings` is embedded in each layout/external layout of the project JSON and will open a PR shortly with a proposal to store them separately.
